src/storage/disk_manager.c

```diff
@@ -2452,72 +2445,6 @@ disk_volume_boot (THREAD_ENTRY * thread_p, VOLID volid, DB_VOLPURPOSE * purpose_
   return error_code;
 }
 
-#if defined (SERVER_MODE)
-int
-disk_auto_expand (THREAD_ENTRY * thread_p)
```

Comment:
Please remove the declaration of it too.

Reply:
There is `disk_auto_expand ()` in storage/disk_manager.h.

Reply:
@wrlawodms
Sorry I didn't see it.